libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorRecognizer.cs

```diff
@@ -41,6 +41,8 @@ public class OrchestratorRecognizer : AdaptiveRecognizer
         /// Property key used when storing extracted entities in a custom event within telemetry.
         /// </summary>
         public const string EntitiesProperty = "entityResult";
+
+        public const string NoneIntentLabel = "None";
```

Comment:
I think we can use the `NoneIntent` declared here in the base class: https://github.com/microsoft/botbuilder-dotnet/blob/402bc02b4cbbd2f4ec359134640e99211367e4a5/libraries/Microsoft.Bot.Builder.Dialogs/Recognizer.cs#L43